integration/messaging/test/static/default-sw/index.html

```diff
@@ -1,17 +1,20 @@
 <html>
-    <head>
-      <title>FCM Demo</title>
-      <meta name="viewport" content="width=device-width,initial-scale=1">
-    </head>
-    <body>
-      <h1>Default SW</h1>
-      <script src="/firebase/firebase-app.js"></script>
-      <script src="/firebase/firebase-messaging.js"></script>
-      <script src="/app.js"></script>
-      <script src="../valid-no-vapid-key/firebaseConfig.js"></script>
-      <script src="https://cdnjs.cloudflare.com/ajax/libs/sinon.js/4.1.3/sinon.min.js"></script>
-      <script>
-        window.__test = new window.DemoApp(window.firebaseConfig);
-      </script>
-    </body>
-    </html>
+  <head>
+    <title>FCM Demo</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+  </head>
+  <body>
+    <h1>Default SW</h1>
+    <script src="https://www.gstatic.com/firebasejs/7.14.0/firebase-app.js"></script>
```

Comment:
Can you reference the local scripts, otherwise we are not really testing the code in the current branch? It also applies to other locations where we reference scripts.

Reply:
Scripts are built and output to the `packages/firebase` folder.